packages/api-client/index.ts

```diff
@@ -378,6 +378,22 @@ class APIClient {
       this.req("PATCH", `/api/v1/alerts/${alertId}`),
   };
 
+  organizations = {
+    addMember: async (userId: number, organizationId: number): Promise<void> =>
```

Comment:
I think that these routes should not Promise void, but maybe return the member id, in which case the front end has the information of the new member created.